.github/shared/src/changed-files.js

```diff
@@ -13,7 +13,7 @@ debug.enable("simple-git");
  * @param {Object} [options]
  * @param {string} [options.baseCommitish] Default: "HEAD^".
  * @param {string} [options.cwd] Current working directory.  Default: process.cwd().
- * @param {string} [options.headCommitish] Default: "HEAD".
+ * @param {string} [options.headCommitish] Use empty string "" to include changes not yet committed. Default: "HEAD".
```

Comment:
I don't think we need to support uncommitted changes yet.  Let's wait until we do, to add this.